src/Components/FirstScreen/Net-component/Net.tsx

```diff
@@ -8,7 +8,7 @@ const NetComponent = () => {
         viewBox="0 0 1440 911"
         fill="none"
         xmlns="http://www.w3.org/2000/svg"
-      >
+      > 
```

Comment:
i would say that svg definitions should be saved as .svg files in Images/ assets folder no having the code in a component defintion

Reply:
Tried to add an svg just like a container background image, but got a bad results -  extra rows appearing. Is it a bad practise to use svg as a React component?